src/com/eviware/soapui/ScenarioReader.groovy

```diff
@@ -197,7 +188,7 @@ class ScenarioReader {
             s.data.put('IDX_READD_PAIR_COUNTS', acrList.join(','))
             s.data.put('idxAdminCodes', indexAdmissionCodes.join(','))
             s.data.put('readdCodes', readmissionCodes.join(','))
-            measureList << s.eval((perfYear == '2018') ? SNIPPETS['2018_ACR_MEASURE'] : SNIPPETS['ACR_MEASURE'])
```

Comment:
Are the 2018 templates not different than the 2017 templates?

Reply:
Yes, when I first updated this template to add all those performance checks, it got integrated back into it, I had to revert it back to the original one without those performance checks.

Reply:
When we create the PR, we have to manually remove the forked binaryrain project before pushing the create PR button or it creates the PR in the binary rain repo instead of Biju's repo.